src/js/modules/facetedsearch/filter-handler.ts

```diff
@@ -3,31 +3,59 @@
  * file that was distributed with this source code.
  */
 import {API} from 'nouislider';
+import getQueryParameters from './urlparser';
 
 export default function (values: Array<string | number>, slider: API) {
   const {prestashop, Theme: {events}} = window;
 
-  const label = slider.target.dataset.sliderLabel;
-  const unit = slider.target.dataset.sliderUnit;
-  const encodedUrl = window.location.href;
-  const splittedUrl = encodedUrl.split('?');
-  let newUrl: string;
+  // Prepare query parameters
+  let queryParams = <any[]> [];
```

Comment:
I think this way should work
```suggestion
  let queryParams = Record<string, any> [];
```